CMake/External_Ceres.cmake

```diff
@@ -26,6 +26,13 @@ else()
   list(APPEND Ceres_EXTRA_BUILD_FLAGS -DMINIGLOG:BOOL=ON)
 endif()
 
+if (fletch_ENABLE_GFlags)
+  list(APPEND Ceres_DEPENDS GFlags)
+  list(APPEND Ceres_EXTRA_BUILD_FLAGS
+         -Dgflags_DIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/lib/cmake/gflags
```

Comment:
My only though for improvement is, see if the gflags_DIR variable that we set in the External_gflags.cmake file is configured correctly. It would be better to use that variable to minimize the changes if gflags changes its installed location.